### ISSUE TITLE
Volume inconsistencies during use

### ISSUE THREAD

**Issue by emily-phet** [phetsims/friction#159]
Issue 1: Many (all?) sounds seem to 'adjust' themselves after load/re-load. Will play at one volume initially and slightly softer after that. Doesn't happen after reset. 

Reproduce 1: On sim load, with mouse, repeatedly pick up zoomed-in chemistry book. First pick up sound is louder than those that follow. (Safari, on my Mac)

Issue 2: After all 'losable' particles have flown off and you're rubbing the final layer on the book, volume seems to get louder (a little too loud). 

If it seems like a more difficult fix, might be best to go ahead and let QA start testing so they can be putting together the full list of challenges. Let me know what you think.

**Comment by zepumph** [comment]
Adding label because of email @emily-phet sent to us saying " I believe the only thing remaining is whether or not John would like to try to address the sound levels issue I brought up before RC. "

@jbphet please remove the label if you would like RC to go out before this issue is completed.

**Comment by jbphet** [comment]
For reference, the version to which @emily-phet is referring is [v1.5.0-dev.23](https://phet-dev.colorado.edu/html/friction/1.5.0-dev.23/phet/friction_en_phet.html).

**Comment by jbphet** [comment]
I've done some investigating, here are my initial responses:

1. The louder initial sound doesn't happen for me on my Win 10 machine running Chrome, but I can reproduce it on the PhET MacBook Pro "Erdos" running OS X 10.11 and Safari 9.0.  I tried a couple of things using that machine, but it's not immediately obvious why this is happening.  There is also a small, brief 'tick' or 'hiss' sound on loading in that configuration that should be addressed.  It's not immediately obvious to me how to fix this, so it may take some time.  I think it would be okay if @zepumph went forward with the RC and I fixed it while that was in progress.
2. The volume of the molecule motion sound is controlled by the equation specified in the design document.  I think what's going on here is that the user can't really get the detachable molecules to full vibration before they break off, so the max volume is only heard after the detachable molecules are all gone and the books are being rubbed together.  I experimented with turning down the overall volume of the sound, and that wasn't bad, but it seemed a bit nicer to me to lower the overall volume and then have the molecule motion sound reach its max volume at about 3/4 the max vibration speed and have it get no louder thereafter.  @emily-phet - I'm happy to try either of these before the RC is created, please let me know if one of the alternatives I've laid out sounds more appealing.

**Comment by jbphet** [comment]
@emily-phet - can you log the version of Mac OS and Safari on which you experienced these issues?

**Comment by emily-phet** [comment]
Mac: 10.13.6 (I haven't updated to mohave yet)
Safari: Version 12.0 (13606.2.11)

**Comment by jbphet** [comment]
In one of the commits listed above I changed the mapping used for the volume of the molecule motion from an exponential to a sigmoid function.  For reference, here is what that mapping function looks like in the Desmos online graphing calculator:

![image](https://user-images.githubusercontent.com/3082831/50577548-9599c580-0de7-11e9-8216-b43e2ab8d553.png)

**Comment by emily-phet** [comment]
Nice! I bet that will help a lot.

**Comment by jbphet** [comment]
I've added a number of changes that should address these problems, most of which are related to the initial muting of sounds by the browser.  I'm not quite finished just yet though, because it seems like some of these changes have had a detrimental effect on the book rubbing sound, so I'll need to address that before submitting this for testing.

**Comment by jbphet** [comment]
Okay, the observations about how this sim behaved on Safari led to a lot of changes due to differences in the way that Chrome, Safari, Edge, and Firefox all handle initial enabling of the sound.  I built a dev version with all the changes - @emily-phet, please try this out and see if it fixes the problems that you noted above and let me know.  If it's all good, @zepumph and I will integrate these changes into the current RC.  Link: https://phet-dev.colorado.edu/html/friction/1.6.0-dev.1/phet/friction_en_phet.html.

**Comment by emily-phet** [comment]
@jbphet I tried these on Chrome and Safari on my laptop - and the inconsistent volumes issue is no longer there. yay! 

I did feel that the rubbing sound (the scratchy sound) feels higher in volume than I remember it being. If you're sure nothing could have changed with that sound, don't worry about changing anything now. If you're unsure, it might be worth a look. From what I remember, the scratchy sound become naturally overwhelmed by the jingling bells sound as you the temp increases, but in this version the rubbing sound still felt quite loud and at the forefront even as the temp got high enough for particles to start flying off.

**Comment by jbphet** [comment]
Interesting that you say that about the volume of the rubbing sound.  I initially thought the same thing, so I compared it side by side to the RC, and it seemed to me like the volume was about the same, but it still felt a bit loud to me (on both versions).  How about we just agree to turn it down by, say ~25%?

**Comment by emily-phet** [comment]
Sounds good to me.

**Comment by emily-phet** [comment]
Ahh - the idea sounds good to me. :)

**Comment by jbphet** [comment]
This change has been made, and I'll make sure it gets into the next RC so we can all review it.

**Comment by jbphet** [comment]
We reviewed the volume change for the rubbing sound during today's sound design meeting, and we're good to go for putting in the next RC.

**Comment by zepumph** [comment]
@jbphet I am happy to cherry-pick when I do the rest of the issues if you would like. Just let me know what shas should be added.

**Comment by jbphet** [comment]
The changes have been rolled into the Friction release branch and should be ready to be included in the next RC.  Over to you @zepumph.

**Comment by zepumph** [comment]
@jbphet I left this issue out of the issues to check on for the RC.2 QA issue. If this should be checked on, please add it to the list of issues to verify in https://github.com/phetsims/QA/issues/276. Otherwise close.

**Comment by jbphet** [comment]
Yes, I do think this should be tested, so I've added it to the list for the RC test.
